dev/reference/tools/sync-diff-inspector/tidb-diff.md

```diff
@@ -0,0 +1,56 @@
+---
+title: TiDB 主从集群的数据校验
+category: tools
+---
+
+# TiDB 主从集群的数据校验
+
+用户可以使用 TiDB-Binlog 搭建 TiDB 的主从集群，Drainer 在把数据同步到 TiDB 时，保存 checkpoint 的同时也会将上下游的 TSO 对应关系保存为 `ts-map`。在 sync-diff-inspector 中配置 `snapshot` 即可对 TiDB 主从集群的数据进行校验。
+
+## 获取 ts-map
+
+在下游 TiDB 中执行以下 SQL 语句：
+
+```
+    mysql> select * from tidb_binlog.checkpoint;
```

Comment:
Please remove the 4 spaces at the beginning of each line, if not necessary.

Reply:
done

Reply:
done